.github/workflows/nodejs.yml

```diff
@@ -97,14 +160,35 @@ jobs:
           --health-cmd="wget localhost:6006 || exit 1" --health-interval=5s --health-retries=10 --health-timeout=2s
 
     steps:
-    - uses: actions/checkout@v2
-    - name: Use Node.js ${{ matrix.node-version }}
-      uses: actions/setup-node@v1
-      with:
-        node-version: ${{ matrix.node-version }}
-        cache: 'npm'
-        cache-dependency-path: '**/package-lock.json'
-    - run: npm install -g npm@7
-    - run: npm ci
-    - run: npm run build
-    - run: npm run test:browser
+      - uses: actions/checkout@v3
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v3
+        with:
+          node-version: ${{ matrix.node-version }}
+
+      - name: Setup npm version 8
+        id: npm8
+        run: |
+          npm i -g npm@7 --registry=https://registry.npmjs.org
+
+      - name: Cache node modules
+        id: cache-nodemodules
+        uses: actions/cache@v2
```

Comment:
This is using `actions/cache@v2`, while the other additions are using `@v3`, is that intentional?